bundles/org.openhab.ui/web/src/pages/home.vue

```diff
@@ -175,6 +174,12 @@ export default {
       this.$store.dispatch('stopTrackingStates')
     },
     onPageInit () {
+      this.$store.subscribe((mutation, state) => {
```

Comment:
@ghys This could also be placed in `homecards-mixin.js` but it is only used by `home.vue` and `home-edit.vue` and I don't think we need this subscription for the edit page. I don`t think a user would want to update the semantic model tags and have the edit page open at the same time.

Do you agree?

Reply:
Sure - makes sense.